packages/automerge-repo-network-broadcastchannel/test/index.test.ts

```diff
@@ -14,4 +15,29 @@ describe("BroadcastChannel", () => {
   }
 
   runAdapterTests(setup)
+
+  it("should allow a channel name to be specified in the options", async () => {
```

Comment:
this is testing two things here -- first that it can set a channel name and second that it doesn't connect. can you either update the test description or split the tests into two? other than that, this is great, ship it

Reply:
Have updated the description here as I am unsure how to test specifically that the name has been set from outside. If we add further options, this could be split out further relatively simply, I would think